src/dev_test.erl

```diff
@@ -84,6 +116,47 @@ postprocess(_Msg, #{ <<"body">> := Msgs }, Opts) ->
     hb_http_server:set_opts(Opts#{ <<"postprocessor-called">> => true }),
     {ok, Msgs}.
 
+%% @doc Find a test worker's PID and send it an update message.
+update_state(_Msg, Msg2, _Opts) ->
+    case hb_ao:get(<<"test-id">>, Msg2) of
+        not_found ->
+            {error, <<"No test ID found in message.">>};
+        ID ->
+            LookupResult = hb_name:lookup({<<"test">>, ID}),
+            case LookupResult of
+                undefined ->
+                    {error, <<"No test worker found.">>};
+                Pid ->
+                    Pid ! {update, Msg2},
+                    {ok, Pid}
+            end
+    end.
+
+%% @doc Find a test worker's PID and send it an increment message.
+increment_counter(_Msg1, Msg2, _Opts) ->
+    case hb_ao:get(<<"test-id">>, Msg2) of
+        not_found ->
+            {error, <<"No test ID found in message.">>};
+        ID ->
+            LookupResult = hb_name:lookup({<<"test">>, ID}),
+            case LookupResult of
+                undefined ->
+                    {error, <<"No test worker found for increment.">>};
+                Pid when is_pid(Pid) ->
+                    Pid ! {increment},
+				    {ok, Pid};
+                _ -> % Handle case where registered value isn't a PID
+                    {error, <<"Invalid registration found for test worker.">>}
+            end
+    end.
+
+%% @doc Does nothing, just sleeps for 3 seconds to simulate a long-running task.
+long_task(_Msg1, _Msg2, _Opts) ->
+    ?event({'dev_test:long_task:sleeping'}),
+    timer:sleep(3000),
```

Comment:
For tests just 750ms or so would work (and save us all hours of waits over the thousands of run `rebar3 eunit` gets 😂)